src/structurizr/mixin/model_ref_mixin.py

```diff
@@ -40,6 +40,11 @@ def model(self) -> "Model":
         """Return the referenced model."""
         return self.get_model()
 
+    @property
+    def is_in_model(self) -> bool:
```

Comment:
Could be 
```suggestion
    def has_model(self) -> bool:
```
but you probably thought of that and prefer this name. Both are fine with me.

Reply:
Nope, I like yours better :)